apps/mark-scan/backend/src/store.ts

```diff
@@ -91,6 +95,48 @@ export class Store {
     return electionRow?.jurisdiction;
   }
 
+  /**
+   * Sets the current precinct `mark-scan` is printing and interpreting ballots for.
+   */
+  setPrecinctSelection(precinctSelection?: PrecinctSelection): void {
+    if (!this.hasElection()) {
+      throw new Error('Cannot set precinct selection without an election.');
+    }
+
+    this.client.run(
+      'update election set precinct_selection = ?',
+      precinctSelection ? JSON.stringify(precinctSelection) : null
+    );
+  }
+
+  /**
+   * Gets the current precinct `scan` is accepting ballots for. If set to
+   * `undefined`, ballots from all precincts will be accepted.
```

Comment:
I was under the impression that the precinct selection had to be the special `AllPrecincts` value for ballots from all precincts to be accepted. I thought that `undefined` just indicated that a precinct hadn't been selected yet. I might be missing something!

Reply:
No I think you're right. I copied this part of the schema from apps/scan, including comment, but an undefined precinct selection might mean something different in `scan`. In `mark-scan` it should mean the same thing as in `mark`, which is how you describe it.

Reply:
Deleted comment on getter: [bef2b8c](https://github.com/votingworks/vxsuite/pull/3767/commits/bef2b8c2333ba9ff8ab7f8e2571f12d74c8e7d79)